src/elements/content-sidebar/Sidebar.js

```diff
@@ -162,6 +169,20 @@ class Sidebar extends React.Component<Props, State> {
         return this.isForced() !== null;
     }
 
+    /**
+     * Helper to navigate sidebar tabs programatically
+     * @param {string} path - The path to navigate
+     * @param {Object} options - Optional options object
+     * @param {boolean} options.replace - Optional flag to replace the current path
+     * @returns void
+     */
+    navigate = (path: string = '/', { replace = false }: { replace: boolean } = {}) => {
+        const { history } = this.props;
+        const method = replace ? history.replace : history.push;
+
+        method(path.charAt(0) === '/' ? path : `/${path}`);
```

Comment:
Are other places calling navigate that don't include a `/`? Both calls above seem to.